cluster/gce/config-test.sh

```diff
@@ -42,19 +42,10 @@ NODE_LOCAL_SSDS=${NODE_LOCAL_SSDS:-0}
 NODE_LABELS="${KUBE_NODE_LABELS:-}"
 WINDOWS_NODE_LABELS="${WINDOWS_NODE_LABELS:-}"
 APISERVERS_EXTRA_NUM=${APISERVERS_EXTRA_NUM:-0}
+APISERVER_DATAPARTITION_CONFIG=${APISERVER_DATAPARTITION_CONFIG:-"abcdefghijklmnopqrstuvwxyz"}
```

Comment:
Can there be a common var file?

Reply:
cluster/gce/config-test.sh and cluster/gce/config-default.sh are the common var files for kube-up and start-kubemark. it is same as /hack/lib/common-var-init.sh.

Reply:
Here cluster/gce/config-default.sh and cluster/gce/config-test.sh share many common variables and have the same value.

Reply:
IIRC they are used in different places. config-default is used in kube-up (admin cluster). config-test is used in start-kubemark (kubemark cluster). they can have different configurations - e.g. 6443 is the api port for admin cluster, 443 is default api port for kubemark iirc.

Reply:
Different places can also share same functional module. I do see many times the two config files had to be changed together for same variables and values. I don't know why they can not be merged.